.github/workflows/check_release_name_bump.yml

```diff
@@ -0,0 +1,41 @@
+name: Check release bump
+
+env:
+  TZ: Europe/London
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  check:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: get release-version
+      run: |
+        echo "RELEASE_VERSION=$(printf '%s\n' $(cat .release-version))" >> $GITHUB_ENV
+    
+    - name: get latest-release
+      id: last_release
+      uses: InsonusK/get-latest-release@v1.0.1
+      with:
+        myToken: ${{ github.token }}
+        exclude_types: "draft|prerelease"
+        view_top: 1
+
+    - name: "Print release info"
+      run: |
+        echo "id: ${{ steps.last_release.outputs.id }}"
+        echo "name: ${{ steps.last_release.outputs.name }}"
+        echo "tag_name: ${{ steps.last_release.outputs.tag_name }}"
+        echo "created_at: ${{ steps.last_release.outputs.created_atd }}"
+        echo "draft: ${{ steps.last_release.outputs.draft }}"
+        echo "prerelease: ${{ steps.last_release.outputs.prerelease }}"
+    
+    - name: "Compare release names"
+      run: |
+        if [["${{ steps.last_release.outputs.tag_name }}" == "${{ env.RELEASE_VERSION }}"]]; then exit 1; fi
```

Comment:
You could just use `$(printf '%s\n' $(cat .release-version))` here instead of adding it to an environmental variable since it's not used anywhere else.

I like this check, short and sweet.